### ISSUE TITLE
What's the status of this project?

### ISSUE THREAD

**Issue by cshaa** [getzze/gconnect#10]
Hello,
if I understand it correctly, this project aims to provide the same functionality as MConnect. However, MConnect seems to be dead by now, the last update came in January 2018. What is the current status of your project? Would you say it's in active development? Is it stable, or still beta? How do you think it compares with MConnect?
Thank you very much for the information :)

**Comment by getzze** [comment]
Hi, I am not using this code anymore and switched to [GSConnect](https://github.com/andyholmes/gnome-shell-extension-gsconnect).
This project is less tested than MConnect (only by me I guess) so it should be less stable and it is not compatible with KDE-connect indicator, so I don't think it's very user friendly.
I tried to mimic as much as possible the code from the original KDE Connect so it would be easier to follow their changes, where MConnect is reimplementing most of the features independently.
Last time I used it, it was stable and the bluetooth was sometimes working, but I don't know after all these years...
